client/src/components/Policy/Policy.vue

```diff
@@ -36,7 +36,7 @@ export default {
   computed: {
     getIntroDt() {
       const dt = new Date(this.date_introduced);
-      return `${dt.getMonth()}-${dt.getDate()}-${dt.getFullYear()}`;
+      return `${dt.getMonth() + 1}-${dt.getDate()}-${dt.getFullYear()}`;
```

Comment:
@markstur ,
This is where the bug was that I mentioned in the other comment (on the "should return formatted date" test). getMonth is 0 indexed so it must be incremented.

I think the fact that this also doesn't take the timezone of the input into account when displaying the date is the what causes the test to fail in different time zones. Maybe this.date_introduced doesn't know it is in GMT and so it gets presented oddly when the tests are ran while the timezones are technically on different days.

I tend to use moment.js for most of my date stuff since it usually handles all of the formatting pretty well.